tests/test_registrations/test_views.py

```diff
@@ -0,0 +1,476 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''Views tests for the OSF.'''
+
+from __future__ import absolute_import
+import datetime as dt
+import mock
+import httplib as http
+from dateutil.parser import parse as parse_date
+
+from nose.tools import *  # noqa PEP8 asserts
+
+from modularodm import Q
+
+from framework.exceptions import HTTPError
+from framework.auth import Auth
+
+from website.models import Node, MetaSchema, DraftRegistration
+from website.project.metadata.schemas import ACTIVE_META_SCHEMAS, _name_to_id
+from website.util import permissions, api_url_for
+from website.project.views import drafts as draft_views
+
+from tests.factories import (
+    NodeFactory, AuthUserFactory, DraftRegistrationFactory, RegistrationFactory
+)
+from tests.test_registrations.base import RegistrationsTestBase
+
+from tests.base import get_default_metaschema
+DEFAULT_METASCHEMA = get_default_metaschema()
```

Comment:
See note above regarding `get_default_metaschema()` call. Hopefully it's not a problem; just a bit surprised it works.